test/integration/sorted-set.test.ts

```diff
@@ -616,6 +616,487 @@ describe('Integration tests for sorted set operations', () => {
+      const sortedSetName = v4();
+
+      beforeAll(done => {
+        const setupPromise = Momento.sortedSetPutElements(
+          IntegrationTestCacheName,
+          sortedSetName,
+          {
+            bam: 1000,
+            foo: 1,
+            taco: 90210,
+            bar: 2,
+            burrito: 9000,
+            baz: 42,
+            habanero: 68,
+            jalapeno: 1_000_000,
+          }
+        );
+        setupPromise.then(() => {
+          done();
+        });
+      });
+
+      it('should fetch only the matching elements if minScore is specified', async () => {
+        const response = await Momento.sortedSetFetchByScore(
+          IntegrationTestCacheName,
+          sortedSetName,
+          {
+            minScore: 100,
+          }
+        );
+
+        expect(response).toBeInstanceOf(CacheSortedSetFetch.Hit);
+        const hitResponse = response as CacheSortedSetFetch.Hit;
+        expect(hitResponse.valueArray()).toEqual([
+          {value: 'bam', score: 1000},
+          {value: 'burrito', score: 9000},
+          {value: 'taco', score: 90210},
+          {value: 'jalapeno', score: 1_000_000},
+        ]);
+      });
+
+      it('should fetch only the matching elements if maxScore is specified', async () => {
+        const response = await Momento.sortedSetFetchByScore(
+          IntegrationTestCacheName,
+          sortedSetName,
+          {
+            maxScore: 1000,
+          }
+        );
+
+        expect(response).toBeInstanceOf(CacheSortedSetFetch.Hit);
+        const hitResponse = response as CacheSortedSetFetch.Hit;
+        expect(hitResponse.valueArray()).toEqual([
+          {value: 'foo', score: 1},
+          {value: 'bar', score: 2},
+          {value: 'baz', score: 42},
+          {value: 'habanero', score: 68},
+          {value: 'bam', score: 1000},
+        ]);
+      });
+
+      it('should fetch only the matching elements if minScore and maxScore are specified', async () => {
+        const response = await Momento.sortedSetFetchByScore(
+          IntegrationTestCacheName,
+          sortedSetName,
+          {
+            minScore: 100,
+            maxScore: 10_000,
+          }
+        );
+
+        expect(response).toBeInstanceOf(CacheSortedSetFetch.Hit);
+        const hitResponse = response as CacheSortedSetFetch.Hit;
+        expect(hitResponse.valueArray()).toEqual([
+          {value: 'bam', score: 1000},
+          {value: 'burrito', score: 9000},
+        ]);
+      });
+
+      it('should fetch an empty list if minScore is out of range', async () => {
+        const response = await Momento.sortedSetFetchByScore(
+          IntegrationTestCacheName,
+          sortedSetName,
+          {
+            minScore: 2_000_000,
+          }
+        );
+
+        expect(response).toBeInstanceOf(CacheSortedSetFetch.Hit);
+        const hitResponse = response as CacheSortedSetFetch.Hit;
+        expect(hitResponse.valueArray()).toEqual([]);
+      });
```

Comment:
@wandaitzuchen - same as previous PR - I don't necessarily need you to review this, but there are 3 tests that I added in this PR that are asserting that we get back a Hit with an empty list of values, but we are actually getting back a Miss.